js/components/pages/PassagePage.react.js

```diff
@@ -51,11 +60,17 @@ export class PassagePage extends Component {
     return (
       <div>
         <div className="mode-controls">
-          <button className="scripture-mode" onClick={() => { dispatch(asyncChangeMode(VERSE_MODE)) }}>Single verse</button>
+          <button className="scripture-mode" onClick={() => { dispatch(asyncChangeMode(VERSE_MODE)) }}>
+            {verses[active[VERSE_MODE]]._verses[0].book.substr(0,3)+' '+formatVerseIndices(verses[active[VERSE_MODE]]._verses[0])}
```

Comment:
I assume you mean unit testing, which isn't something I'm very familiar with. Please feel free to do whatever you feel is best in this case.

I see what you mean by the _verses being private. Normally I would just add a public verses var that returns the current value of _verses. I think that's what you were getting at, so do you think that's the best way forward?